### ISSUE TITLE
Material date picker is not focusable with a keyboard

### ISSUE THREAD

**Issue by gmoraja** [flutter/flutter#49809]
@goderbauer
@gspencergoog
@lorenzOliveto

**Comment by darrenaustin** [comment]
The following PRs complete the support for keyboard navigation in the Material Date Pickers:

#59279
#59310
#59586
#60497 

To traverse the date pickers you can use the standard Tab/Shift-Tab to navigate between controls of the pickers. However, when you get to the day grid, tab will only traverse into and out of the grid. To navigate the keyboard focus within the grid you can use the following:

Key | Action
------------ | -------------
Tab | move focus to next element outside grid
Shift-Tab | move focus to previous element outside grid
Left Arrow | move focus to previous day
Right Arrow | move focus to next day
Up Arrow | move focus to previous week
Down Arrow | move focus to next week
Enter / Space | select the currently focused day

**Comment by gmoraja** [comment]
@darrenaustin thanks for working on this. Everything seems to work fine, except the fact that the focus doesn't move out of the "minutes" field. Using the Tab key creates extra space inside the field, instead of moving the focus to the next element. If needed, I can upload a video to show you the behavior. Should I open a new issue or continue the conversation on this thread?
